be/src/olap/tablet_meta.h

```diff
@@ -32,21 +32,18 @@ namespace doris {
 // Class for managing tablet header.
 class TabletMeta : public OLAPHeaderMessage {
```

Comment:
OLAPHeaderMessage  should be renamed?

Reply:
TabletMeta will not inherit from protobuf in the future. 
OLAPHeaderMessage will be replace by a new pb, and is a member of TabletMeta.